src/utils/__tests__/generateChartData.js

```diff
@@ -0,0 +1,113 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import generateChartData from "../generateChartData";
+import { RELEASES } from "../../constants/metrics";
+
+describe("generateChartData.js", () => {
```

Comment:
Given the density of the logic, let's add some more test cases for this function. Here are some ideas:
- When the `data` arg is empty
- When the `data` arg has more elements than the `period` arg
- When the `metrics` arg has more than one metric
- When the `metrics` arg is empty (error?)

Reply:
@jessex Added a bunch of tests